scripts/shared.sh

```diff
@@ -39,6 +39,22 @@ function get_tfs_executable() {
     rm ${zip_file} && rm -rf exec_dir
 }
 
+function get_device_type() {
+    if [ $1 = 'gpu' ]; then
+        echo 'gpu'
+    else
+        echo 'cpu'
+    fi
+}
+
+function get_repository_name() {
+    if [ $1 = 'ei' ]; then
+        echo 'sagemaker-tensorflow-serving-eia'
+    else
+        echo 'sagemaker-tensorflow-serving'
+    fi
+}
+
 function parse_std_args() {
     # defaults
     arch='cpu'
```

Comment:
this is renamed to 'device' everywhere else. let's use one or the other not both.

Reply:
To match the EI image name format of other frameworks, the image name for TFS is:
sagemaker-tensorflow-serving-eia:1.12.0-cpu

So we need both $arch and $device variables. We could have user pass 'eia' as $arch, but the tag still uses 'cpu'

Reply:
It probably would have made more sense to treat eia the way we do cpu and gpu, and put it in the same repo. But that would impact other projects and is way out of scope for this pr :)